template/7-functional_requirements.md

```diff
@@ -1,22 +1,21 @@
 # Functional Requirements
 
-*Max 3 pages.*
+1. **User Authentication**: Users must be able to authenticate into their account to user the app. For this, we use pre-existing infrastructure, such as google authentication.
 
-*List the key features of the MVP precisely.*
+2. **Post Creation**: Users must be able to create posts containing text. These posts will have embedded location data.
```

Comment:
What about other forms of media like pictures and gifs. I think they are important to have for the MVP.